source/clear-linux/reference/collaboration/structure-formatting.rst

```diff
@@ -0,0 +1,483 @@
+websites:
+
+* `Sphinx documentation`_
+* `reStructuredText Primer`_
+
+You can view the content directly in the .rst markup files, or you can generate
+the HTML content by installing and running the documentation locally. To run the
+documentation locally, follow the instructions found in the
+`documentation repository`_ README.
+
+New pages
+=========
+
+There are a few additional steps to consider when adding a new page to the
+documentation. First, identify where your new page should be located within the
+existing `Documentation organization`_. Second, make sure the new page is picked
+up in the Sphinx build and easily linkable from other content.
+
+Each page must be included in a `Sphinx toctree`_ in order to be included in the
+documentation content tree. Typically, pages are added to the section landing
+page toctree.
+
+For example, the :ref:`collaboration` page toctree looks like:
+
+.. code-block:: rest
+
+   .. toctree::
+      :maxdepth: 1
+
+      writing-guide
+      structure-formatting
+
+Additionally, each page must include a uniquely named reST label directly before
+the page title, to enable the `Sphinx ref role`_ for linking to a page.
+
+For example, this page "Structure and formating" has the label
+``.. _structure-formatting``:
+
+.. code-block:: rest
+
+   .. _structure-formatting:
+
+   Structure and formatting
+   ########################
+
+This page can then be referenced from other pages in the documentation using the
+`:ref:` role:
+
+.. code-block:: rest
+
+   :ref:`structure-formatting`
+
+Documentation organization
+**************************
+
+The documentation is organized into five general sections:
+
+#. **Concepts**: Introduction and overview of |CL| specific concepts or
+   features.
+#. **Get started**: Information about getting started with |CL|.
+#. **Guides**: Detailed information and instruction on using |CL| features.
+#. **Tutorials**: Step-by-step instruction for using |CL| in specific use cases.
+#. **Reference**: Supplementary and reference information for |CL|.
+
+Page structure
+==============
+
+Each page in the documentation should follow the basic format of:
+
+* Overview: 1-2 sentences describing what this page shows and why it matters
+* Prerequisites: Describe any pre-work necessary to the content (if appropriate)
+* Content
+* Next steps: List links to next steps (if appropriate)
+* Related topics: List links to related content (if appropriate)
+
+Headings
+========
+
+Use headings to section and organize your content for better readability and
+clarity.
+
+* All files must have a top level heading, which is the title for the page.
+* Headings should be short and descriptive.
+* Write headings in sentence case (capitalize only the first word and proper
+  nouns).
+* Up to three additional levels of headings are allowed under the title heading.
+* Each heading must be followed by at least one paragraph of content. There
+  should never be two consecutive headings.
+

```

Comment:
Add an example:  
**Use this:**
Install software

**Not this:**
Installing the Software on the Platform

Reply:
Great example. I added this example and moved the grammar/style related bullets into the writing guide section on headings. (trying to keep style/grammar in one doc, structure/formatting in the other).

Reply:
Your rewrites are even better - LGTM.